horizomer/benchmark/parse_output.py

```diff
@@ -122,12 +122,12 @@ def parse_darkhorse(input_f, output_fp, low_lpi=0.0, high_lpi=0.6):
     hgts = []
     # skip header
     next(input_f)
-    for l in input_f:
-        l = l.strip('\r\n').split('\t')
-        best_hit_ids.add(l[3])
-        if low_lpi < float(l[5]) < high_lpi:
-            hgt = '\t'.join((l[0], l[3], l[12], l[13], l[14],
-                             l[6], l[9], l[4]))
+    for line in input_f:
```

Comment:
I personally don't like those magic numbers. From just looking at the code, I figure you could also read this file in via Pandas.read_csv(..., sep='\t') and than select specific columns by their header names or assign header names.

Reply:
Those are some very old codes (not sure if coded by me). The reason you see those in this PR is because the latest PEP8 complains about the usage of `l`, since one can confuse it with `1`. Therefore, I had to replace all `l`s with `x`s.

Reply:
lol, as if x would be that much better. How about "fields" or "columns"? I foresee PEP complaining about single char variables in the future.

Reply:
I feel that `fields` or `columns` are too long are temporary variable names. single-letter names like `x` have been used extensively by people already, such as `[x.name for x in tree.tips()]`. People will complain the PEP8 prohibits this (well people are already complainin about PEP8 itself).

Reply:
whatever, this is absolutely not blocking. Ignore this discussion!